src/nft/components/details/detailsV2/NftDetails.tsx

```diff
@@ -8,11 +9,29 @@ interface NftDetailsProps {
   collection: CollectionInfoForAsset
 }
 
+const DetailsBackground = styled.div<{ backgroundImage: string }>`
+  position: fixed;
+  top: ${({ theme }) => `-${theme.navHeight}px`};
```

Comment:
i feel like negative spacing can lead to unexpected problems - any other way to do this?

Reply:
forgot that since this is `fixed` it doesn't even need the spacing